packages/ic-response-verification-tests/Cargo.toml

```diff
@@ -6,8 +6,8 @@ edition = "2021"
 [dependencies]
 hex = "0.4.3"
 http = "0.2.8"
-ic-agent = "0.23.1"
-ic-utils = "0.23.1"
+ic-agent = { git = "https://github.com/dfinity/agent-rs.git" }
+ic-utils = { git = "https://github.com/dfinity/agent-rs.git" }
```

Comment:
There was a bug with `ic-utils`. This is affecting the e2e tests and ICX Proxy.

https://github.com/dfinity/agent-rs/pull/432

Reply:
Can we pin here to a specific commit hash?

Reply:
Cargo is doing this automatically: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories
> Once a git dependency has been added, Cargo will lock that dependency to the latest commit at the time. New commits will not be pulled down automatically once the lock is in place. However, they can be pulled down manually with cargo update.

And I hope to update it this week anyway once `agent-rs` does another release.